modules/custom/d_p_side_tiles/templates/paragraph--d-p-side-tiles.html.twig

```diff
@@ -69,14 +69,14 @@
     <div {{ attributes.addClass(classes) }}>
       <div {{ tiles_wrapper_attributes.addClass(tiles_wrapper_classes) }}>
         <div {{ images_side_attributes.addClass(images_side_classes) }}>
-          {{ content.group_d_side_tiles_gallery }}
+          {{ content.group_d_tiles_content.field_d_media_image }}
```

Comment:
Generally, we wanted to avoid referring to fields here, that is why we introduced groups. So this `{{ content.group_d_side_tiles_gallery }}` was ok, but it didn't work